### PR TITLE
Revisit the summary format and structure

### PR DESCRIPTION
**Pull request by gdelgadot**
Signed-off-by: Gloria Delgado <gloriad@vmware.com>

### CODE REVIEW

**Comment by gdelgadot** [comment]
@recena something like this? Do you want to remove the actions summary of the last line?
[Test repo](https://github.com/gdelgadot/prueba-action/runs/7468074985?check_suite_focus=true)
Screenshot 2022-07-22 at 14 25 20

File: src/main.ts [hunk 1]
```diff
@@ -392,22 +392,35 @@ export function prettifyExecutionGraphResult(
...
           ansi.red(task["tests"]["failed"])
         )} ${ansi.bold(ansi.red(" failed"))}`
       )
-    } else if (task["passed"] === true) {
-      core.info(ansi.bold(`${task["action_id"]}: ${ansi.green("passed")}`))
-    } else if (task["passed"] === false) {
-      core.info(ansi.bold(`${task["action_id"]}: ${ansi.red("failed")}`))
-    }
-    if (task["vulnerabilities"]) {
+    } else if (task["passed"] === true && task["vulnerabilities"]) {
```
Comment (recena) [inline src/main.ts:405]: Why `vulnerabilities` is a task? What we have is the action `trivy`

**Comment by recena** [comment]
> Do you want to remove the actions summary of the last line?

I don't want, I think it can be helpful.

**Comment by recena** [comment]
@gdelgadot Between the action id (for example: `trivy`) and the term _action_, we need to make sure there is a space.

**Comment by recena** [comment]
@gdelgadot Could you add an example with more actions? cypress or resilience test or health-check, etc...

File: src/main.ts (hunk 1 above)
Comment (gdelgadot) [reply src/main.ts:405]: this was already like it is before last changes. If tasks have vulnerabilities, they are shown that way.

Comment (recena) [reply src/main.ts:405]: Got it.

File: src/main.ts [hunk 2]
```diff
@@ -392,22 +392,35 @@ export function prettifyExecutionGraphResult(
   for (const task of executionGraphResult["actions"]) {
     if (task["tests"]) {
       core.info(
-        `${ansi.bold(task["action_id"])}: ${ansi.bold(
+        `${ansi.bold(task["action_id"])} ${ansi.bold(" action:")} ${ansi.bold(
```
Comment (recena) [inline src/main.ts:395]: In fact, `ansi.bold(" action: ")`, no?

File: src/main.ts [hunk 3]
```diff
@@ -392,22 +392,35 @@ export function prettifyExecutionGraphResult(
...
       )
-    } else if (task["passed"] === true) {
-      core.info(ansi.bold(`${task["action_id"]}: ${ansi.green("passed")}`))
-    } else if (task["passed"] === false) {
-      core.info(ansi.bold(`${task["action_id"]}: ${ansi.red("failed")}`))
-    }
-    if (task["vulnerabilities"]) {
+    } else if (task["passed"] === true && task["vulnerabilities"]) {
+      core.info(
+        `${ansi.bold(task["action_id"])} ${ansi.bold(" action:")} ${ansi.green(
```
Comment (recena) [inline src/main.ts:407]: It is easier to understand `ansi.bold(" action: ")` and.. (next comment)

File: src/main.ts [hunk 4]
```diff
@@ -392,22 +392,35 @@ export function prettifyExecutionGraphResult(
...
+          " passed"
```
Comment (recena) [inline src/main.ts:408]: `passed`

File: src/main.ts [hunk 5]
```diff
@@ -392,22 +392,35 @@ export function prettifyExecutionGraphResult(
...
+        )} ${ansi.bold(ansi.red(" critical"))}, ${
+          task["vulnerabilities"]["unknown"]
+        } unknown`
+      )
+    } else if (task["passed"] === false && task["vulnerabilities"]) {
       core.info(
-        `${ansi.bold("Vulnerabilities:")} ${
+        `${ansi.bold(task["action_id"])} ${ansi.bold(" action:")} ${ansi.red(
+          ansi.bold(" failed")
+        )} » ${ansi.bold("Vulnerabilities:")} ${
```
Comment (recena) [inline src/main.ts:423]: `ansi.bold("Vulnerabilities: ")`

File: src/main.ts (hunk 2 above)
Comment (gdelgadot) [reply src/main.ts]: done

File: src/main.ts (hunk 4 above)
Comment (gdelgadot) [reply src/main.ts]: done

**Comment by gdelgadot** [comment]
@recena what do you think?
[Test with more actions
Screenshot 2022-07-22 at 18 34 27
](https://github.com/gdelgadot/prueba-action/runs/7471831602?check_suite_focus=true)

**Comment by recena** [comment]
@gdelgadot It looks great, just a comment:

```
Pipeline result: failed
trivy action: failed » Vulnerabilities: 0 minimal, 435 low, 150 medium, 133 high, 34 critical, 0 unknown
grype action: failed » Vulnerabilities: 240 minimal, 135 low, 134 medium, 121 high, 35 critical, 16 unknown
health-check action:  » Tests: 1 passed, 0 skipped, 0 failed
goss action:  » Tests: 21 passed, 0 skipped, 0 failed
resilience-tests action:  » Tests: 1 passed, 0 skipped, 0 failed
cypress action:  » Tests: 9 passed, 0 skipped, 0 failed
Actions: 5 passed, 0 skipped, 2 failed, 7 total
```
I miss the result (true | false) after the actions `health-check`, `resilience-tests action` and `cypress action`.

**Comment by recena** [comment]
After the latest changes...

Screenshot 2022-07-28 at 00 21 50

**Comment by mpermar** [comment]
Looks good but you have the double spaces problem. Gloria did fight a bit with that and at some point I think she got it right but never understood why.

**Comment by recena** [comment]
@mpermar Now, it is fixed. You can see the result [here](https://github.com/recena/gh-action-testing/runs/7554860249?check_suite_focus=true).

**Review by mpermar — APPROVED** [review]

**Pull request merged**